projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/ai-assistant/with-ai-assistant.tsx

```diff
@@ -48,6 +46,13 @@ export const withAIAssistant = createHigherOrderComponent(
 
 		const requestSuggestion = useCallback(
 			( promptType: PromptTypeProp, options: AiAssistantDropdownOnChangeOptionsArgProps ) => {
+				const clientIds = select( 'core/block-editor' ).getSelectedBlockClientIds();
+				const blocks = select( 'core/block-editor' ).getBlocksByClientId( clientIds );
+				const content = blocks
+					.filter( block => block != null ) // Safeguard against null or undefined blocks
```

Comment:
What breaks when you remove this? I don't think blocks can be null or undefined?